Files/Views/LayoutModes/GenericFileBrowser.xaml

```diff
@@ -147,6 +147,15 @@
                             <FontIcon  Glyph="&#xE179;" />
                         </MenuFlyoutItem.Icon>
                     </MenuFlyoutItem>
+                    <MenuFlyoutItem
+                    x:Uid="BaseLayoutContextFlyoutColumn"
```

Comment:
Can you add this string to the en-US resource file?

Reply:
done

Reply:
@yaichenbaum